actions/install/installer.sh

```
@@ -0,0 +1,94 @@
+#!/bin/bash
+# SLSA verifier install script
```

Always add `set -euo pipefail` otherwise errors are silently ignored